docs/basics/101-115-symlinks.rst

```diff
@@ -175,30 +175,30 @@ will take a closer look at that.
 .. _objecttree:
 .. index::
    pair: key; git-annex concept
-.. find-out-more:: more about paths, checksums, object trees, and data integrity
+.. find-out-more:: Data integrity and annex keys
    :name: fom-objecttree
 
    So how do these cryptic paths and names in the object tree come into existence?
    It's not malicious intent that leads to these paths and file names - its checksums.
 
-   When a file is annexed, git-annex generates a *key* (or :term:`checksum`) from the **file content**.
+   When a file is annexed, git-annex typically generates a *key* (or :term:`annex key`) from the **file content**.
    It uses this key (in part) as a name for the file and as the path
    in the object tree.
    Thus, the key is associated with the content of the file (the *value*),
-   and therefore, using this key, file content can be identified --
-   or rather: Based on the keys, it can be identified whether file content changed,
-   and whether two files have identical contents.
+   and therefore, using this key, file content can be identified.
 
-   The key is generated using *hashes*. A hash is a function that turns an
-   input (e.g., a PDF file) into a string of characters with a fixed length based on its contents.
+   Most key types contain a :term:`checksum`. This is a string of a fixed number of characters
+   computed from some input, for example the content of a PDF file,
+   by a *hash* function.
 
-   Importantly, a hash function will generate the same character sequence for the same file content, and once file content changes, the generated hash changes, too.
+   This checksum *uniquely* identifies a file's content.
+   A hash function will generate the same character sequence for the same file content, and once file content changes, the generated checksum changes, too.
    Basing the file name on its contents thus becomes a way of ensuring data integrity:
-   File content cannot be changed without git-annex noticing, because file's hash, and thus its key in its symlink, will change.
-   Furthermore, if two files have identical hashes, the content in these files is identical.
+   File content cannot be changed without git-annex noticing, because file's checksum, and thus its key in its symlink, will change.
```

Comment:
```suggestion
   File content cannot be changed without git-annex noticing, because the file's checksum, and thus its key in its symlink, will change.
```